### ISSUE TITLE
The added layer must be instance of the class

### ISSUE THREAD

**Issue by Ritesh-pandey-crypto** [keras-team/keras#14138]
model = Sequential()
model.add(Embedding(top_words, 32, input_length=max_words))
model.add(Flatten())
model.add(Dense(250, activation='relu'))
model.add(Dense(1, activation='sigmoid'))
model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])
print(model.summary())

**Comment by sushreebarsa** [comment]
@Ritesh-pandey-crypto  Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!

**Comment by Ritesh-pandey-crypto** [comment]
Alright, If i face the same issue again, I'll contact you for sure


With Regards,
Ritesh Pandey
